api/logic/layers.go

```diff
@@ -30,7 +30,7 @@ import (
 
 // POSTLayersParameters represents the expected parameters for POSTLayers.
 type POSTLayersParameters struct {
-	ID, Path, ParentID string
+	ID, Path, ParentID, ImageFormat string
```

Comment:
API doc about [layer insertion](https://github.com/coreos/clair/blob/master/docs/API.md#insert-a-new-layer) should be updated as a new parameter appeared